hack/build-marker-podman.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [ -z "$ARCH" ] || [ -z "$PLATFORMS" ] || [ -z "$MARKER_IMAGE_TAGGED_1" ]; then
+    echo "Error: ARCH, PLATFORMS, and MARKER_IMAGE_TAGGED_1 must be set."
+    exit 1
+fi
+
+IFS=',' read -r -a PLATFORM_LIST <<< "$PLATFORMS"
+
+# Remove any existing manifests and images
+podman manifest rm "${MARKER_IMAGE_TAGGED_1}" || true
+podman manifest rm "${MARKER_IMAGE_TAGGED_2}" || true
+podman rmi "${MARKER_IMAGE_TAGGED_1}" || true
+podman rmi "${MARKER_IMAGE_TAGGED_2}" || true
+
+podman manifest create "${MARKER_IMAGE_TAGGED_1}"
+
+for platform in "${PLATFORM_LIST[@]}"; do
+    podman build \
+        --no-cache \
```

Comment:
why no cache ?
speed is important please

Reply:
Removed.